src/ActionModal/components/ActionModal.vue

```diff
@@ -758,6 +765,14 @@ export default {
             }
           )
         }
+        if (this.network.network_type === "polkadot") {
+          transactionData = {
+            fee: {
+              amount: this.polkadotFee,
```

Comment:
```suggestion
              amount: this.estimatedFee,
```

Reply:
The problem is that in `signStep` this is undefined. This is actually the first thing I tried. And then I had to add `signStep` to the `v-if` because of that

Reply:
I see. the computed would trigger and return undefined.